charts/metering-ansible-operator/templates/crds/reportquery.crd.yaml

```diff
@@ -23,3 +23,98 @@ spec:
   - name: Age
     type: date
     JSONPath: .metadata.creationTimestamp
+
+  validation:
+    openAPIV3Schema:
+      type: object
+      required: ["spec"]
+      properties:
+        spec:
+          type: object
+          required: ["columns", "query"]
```

Comment:
The current CRD's that use openAPI expand the `required` array to have a `property` per line, i.e.:
```
required:
- columns
- query
```

Minor nit, but I see there's no real convention on this (as k8s uses the `required: ["columns", ...]` format), and we already use this formatting in existing CRDs.

Reply:
Seems folks are using both ways, but I'll match the convention we have in there already. More cohesive that way.